src/custom/state/claim/hooks/index.ts

```diff
@@ -279,6 +279,17 @@ export function useUserClaims(account: Account, optionalChainId?: SupportedChain
   return { claims: claimKey ? claimInfo[claimKey] : null, isLoading }
 }
 
+let FETCH_DEPLOYMENT_TIMESTAMP_PROMISE: Promise<number> | null = null
```

Comment:
why CAPITAL letters if its not a constant?

Reply:
The same thing they used here https://github.com/gnosis/cowswap/blob/1c086d45eb314c1d21872c2e6f94e06923544552/src/state/claim/hooks.ts#L28

Reply:
well, it sounds like also bad there :) 

anyways, just a nit. Constants are capitalised, but this is not one